app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -3398,6 +3399,24 @@ class PluginApi {
     registerRichEditorExtension(extension);
   }
 
+  /**
+   * Registers a property that will be included when saving a category.
+   *
+   * This is useful for plugins that are adding additional parameters to the category
+   * and want to save the new property alongside the default category properties
+   * (all under the same save call), and are not using custom fields for that.
+   *
+   * ```
+   * api.registerCategoryUpdateProperty("property_one");
+   * api.registerCategoryUpdateProperty("property_two");
+   * ```
+   *
+   * @param {string} property - The name of the property to include when saving a category.
+   */
+  registerCategoryUpdateProperty(property) {
```

Comment:
Debating on whether the name is good enough... we could also go with `registerCategoryProperty`, because it's on initial save and update that this applies, right?